### PR TITLE
Fix Status Handling and License Headers

### PR DESCRIPTION
**Pull request by DomFleischmann**
- Fix Status Handling by always calling main and adding checks
- Add license headers and checks
- Fix charmcraft parts issue

### CODE REVIEW

File: operators/tensorboard-controller/metadata.yaml [hunk 1]
```diff
@@ -1,8 +1,6 @@
-# Copyright 2021 dom
```
Comment (DnPlas) [inline operators/tensorboard-controller/metadata.yaml:1]: Hehe

File: operators/tensorboard-controller/tox.ini [hunk 2]
```diff
@@ -19,6 +25,7 @@ commands =
 deps =
 	black==20.8b1
 	flake8
+	flake8-copyright<0.3
```
Comment (DnPlas) [inline operators/tensorboard-controller/tox.ini:28]: Let's keep consistency, I've seen you've listed these deps in `test-req...txt` [here](https://github.com/canonical/kubeflow-volumes-operator/pull/8) and [here](https://github.com/canonical/admission-webhook-operator/pull/8).

**Review by DnPlas — APPROVED** [review]
LGTM, just a minor comment.

File: operators/tensorboard-controller/tox.ini (hunk 2 above)
Comment (DomFleischmann) [reply operators/tensorboard-controller/tox.ini]: Yes you are right, fixed it.

**Pull request merged**